replit/rippkgs
# CI

Why
===

initialize more comprehensive CI before sharing this repo with the communities

What changed
============

- added nix flake checks:
  - both of the binaries build
  - clippy
  - `cargo fmt`
  - `cargo nextest` (faster than `cargo test`)
  - check nix formatting (still using `alejandra`)
- use github actions instead of semaphore (it's more accessible for our open-source repos + DeterminateSystems easily integrates)
- formatted according to the updated checks

Test plan
=========

n/a
